image-search/Models/Album.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  Album.swift
+//  image-search
+//
+//  Created by Renato Vieira on 08/10/19.
+//  Copyright © 2019 Stant. All rights reserved.
+//
+
+import Foundation
+
+struct SearchImages: Codable{
+
+    let id         : String?
```

Comment:
I'm asking because I'm not sure.. Is this id a String? I thought it was an integer.

Reply:
The id field in api is constructed  with alphanumeric characters!!

Example: {"id":"fa1f6635-8fe3-4261-bbcb-62b30fdfc6b9","artist_name":"Núbia Batista","record_label":"Melo S.A.","album_name":"dolorum non voluptatem","album_cover":"https://picsum.photos/200"}